src/main/java/org/elasticsearch/action/admin/indices/segments/IndicesSegmentsRequest.java

```diff
@@ -37,7 +37,7 @@ public IndicesSegmentsRequest() {
 
     public IndicesSegmentsRequest(String... indices) {
         super(indices);
-        indicesOptions(IndicesOptions.fromOptions(false, false, true, false));
+        indicesOptions(IndicesOptions.fromOptions(false, true, true, false));
```

Comment:
this change affects also the default behaviour of indices segments api. I am not sure we should change that as well? Could you double check what is other apis default behaviour and see if only the cat variant sets different indices options to its own internal request, or if indices segments has a weird default indices options?